registration_form_with_db/templates/register.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Регистрация пользователя</title>
+    <link rel="stylesheet" href="{{ url_for('static', filename='css/bootstrap.min.css') }}">
+    <link rel="stylesheet" href="{{ url_for('static', filename='css/style.css') }}">
+</head>
+<body class="container mt-5">
+    <h2>Форма регистрации</h2>
+    {% with messages = get_flashed_messages() %}
+        {% if messages %}
+            <div class="alert alert-{{ messages[0].category }}">
+                {{ messages[0] }}
+            </div>
```

Comment:
Если у вас будет больше flash сообщений чем одно, то лучше сделать так. Впрочем и для одного будет работать
```suggestion
            {% for message in messages %}
                <div class="alert alert-{{ messages.category }}">
                    {{ message }}
                </div>
            {% endfor %}
```